### ISSUE TITLE
Relativity Core Event Restructure

### ISSUE THREAD

**Issue by jeppy64** [phsc-edu/2018#255]
Re-architect core event entity to use newest Drupal 8.5 date/time storage on Events, Event Days, Time Slots and Department Hours and deprecate old date/list text configs. On each site, once completed, all past events legacy field data will need to be migrated to new field storage. All existing event display views to be updated with newest relationships and architecture.

The following, existing tickets are all part of this initiative:
Ticket #122 - Completed
Ticket #177 
Ticket #209 - Completed
Ticket #210 
Ticket #230 

Note: Currently, all sites "passed events" are still published, but are removed from public site displays. But, these event nodes have been crawled and index by search engines. We should come up with a strategy on all sites for core events on how to handle. ex; investigate an auto-purge routine to delete these after a specific time frame age. All sites upcoming events will be migrated to new architecture.

Existing time estimate was at 60 hours, plus 30 hours for Events Interface consolidation. If we do not have to include passed events in the migrations, we can adjust this estimate down to 70 hours total

**Comment by C13L0** [comment]
@jeppy64 We will not include past events as this only causes issues with the search and google index. Requesting 70 hours...

**Comment by C13L0** [comment]
@jeppy64 Dr. Harts has approved for 70 hours for this ticket. We have tagged this as a priority 1, meaning it is top priority.

**Comment by C13L0** [comment]
From: Information Center
Sent: Monday, July 16, 2018 11:57:44 AM
To: Webmaster
Subject: Re: Updated Events!
 
Cielo,

Here is our feedback:

 
 
The only thing I'm noticing with the events...when I spoke with Cielo the other day she said there would be a 'publish' button that we would need to click on. I saw the button while I was talking to her, and now I don't see it. I'm not sure if it was something glitchy that they were able to change between then and now.
 
 
 
I think that the design is very streamlined and easier to use in that we can go down the list of each event item one by one instead of having to tab over from description to test to contact, where things may be missed. I also like that we can type in times to the exact minute and note changes in the revision log message box.
 
 
 
I think that it is very efficient that we do not have to type in the campus address every time and that the map widget populates automatically. However, it seems to be having the same problem as before, where the Google map is not pulling up the entire campus but redirecting to a particular building (For example, West Campus directs to C building regardless of where the event is held). We solved this problem previously by typing in “Pasco-Hernando State College,” before the street address.
 
 
 
Is there a way to omit the seconds in the time? 
Why must we enter an end date when we have an end time? 
 

 
Thanks!
